src/_prebid.jsx

```diff
@@ -14,24 +14,39 @@ const propTypes = {
   dimensions: PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.number)).isRequired,
   className: PropTypes.string,
   skipIf: PropTypes.func,
-  adserverTimeout: PropTypes.number,
   prebidLibURL: PropTypes.string,
-  config: PropTypes.shape({
-    pricing: PropTypes.arrayOf(PropTypes.shape({
-      min: PropTypes.number,
-      max: PropTypes.number,
-      increment: PropTypes.number,
-      precision: PropTypes.number,
-    })),
+
+  // pbjs setConfig props
+  bidderTimeout: PropTypes.number,
+  currency: PropTypes.shape({
+    adServerCurrency: PropTypes.string,
+    granularityMultiplier: PropTypes.number,
+    conversionRateFile: PropTypes.string,
+    rates: PropTypes.object,
+    bidderCurrencyDefault: PropTypes.object,
   }),
+  debug: PropTypes.bool,
+  priceGranularity: PropTypes.oneOfType([
+    PropTypes.oneOf(['low', 'medium', 'high', 'auto', 'dense']),
+    PropTypes.shape({
+      buckets: PropTypes.arrayOf(PropTypes.shape({
+        min: PropTypes.number,
+        max: PropTypes.number,
+        increment: PropTypes.number,
+        precision: PropTypes.number,
+      })),
+    }),
+  ]),
```

Comment:
I think this is wrong... priceGranularity could be any of those strings in that array (line#30) or the object with that shape (line#31):
```js
priceGranularity: PropTypes.oneOf([
  'low', 'medium', 'high', 'auto', 'dense',
  PropTypes.shape({
    buckets: PropTypes.arrayOf(PropTypes.shape({
      min: PropTypes.number,
      max: PropTypes.number,
      increment: PropTypes.number,
      precision: PropTypes.number,
    })),
  }),
]);
```

Reply:
`PropTypes.oneOf` is for constants, this way is telling it is allowed those strings or `PropTypes.shape...` as a value